pepper-apis/config/manifest.rb

```diff
@@ -15,5 +15,4 @@
 copy_secret_from_path "secret/pepper/#{$env}/pepper-cert.crt"
 render "ddp.conf.ctmpl"
 copy_file "post_deploy_smoketest.sh"
-copy_file "waf.conf"
```

Comment:
There are probably other things in `manifest.rb` that are a distraction, but this PR is just focused on tcell.